lib/cl/thrift.asd

```diff
@@ -34,6 +34,7 @@
  services protocol."
   :serial t
   :components ((:file "package")
+	       (:file "util")
```

Comment:
use split-sequence for that 

(add split-sequence to bundled libraries if not already there)